Brewfile

```diff
@@ -1,8 +1,11 @@
-# Brewfile for installing Honeycomb prerequisites
-# Visit https://brown-ccv.github.io/honeycomb-docs/docs/quick_start for info
-
-brew "git"
-brew "node"
-cask "electron"
-brew "firebase-cli"
-cask "visual-studio-code"
+# Brewfile for installing Honeycomb prerequisites
+# Visit https://brown-ccv.github.io/honeycomb-docs/docs/quick_start for info
+
+brew "git"
+brew install --cask "github"
+brew install "node@18"
+brew install --cask "oracle-jdk"
+brew install --cask "visual-studio-code"
+

```

Comment:
No need for firebase-cli? Or did you move it as a node dee dependency?

Why do we need JDK?

Reply:
Yeah, it's an `npm i -g` install now. That's the only way to get it on Windows so I figured we'd keep it the same for both.

The JDK is for the Firestore Emulators.  It runs some java based backend when using the emulators locally